src/infra/messaging-outbox/sqlite/src/repos/sqlite_outbox_message_repository.rs

```diff
@@ -108,12 +110,13 @@ impl OutboxMessageRepository for SqliteOutboxMessageRepository {
 
             use sqlx::Row;
             let mut query_stream = query.try_map(|event_row: SqliteRow| {
-                Ok(OutboxMessage {
-                    message_id: OutboxMessageID::new(event_row.get(0)),
-                    producer_name: event_row.get(1),
-                    content_json: event_row.get(2),
-                    occurred_on: event_row.get(3),
-                })
+                println!("event_row");
```

Comment:
Oh, good old-fashioned debugging through prints :)

Btw, I advise you to use `dbg!()`, since they are caught via clippy -- I've added this rule, since I sometimes forgot to remove them (debug macros) myself.